ui/src/components/Model.js

```diff
@@ -55,7 +55,11 @@ export default ({ modelName }) => (
   <ModelQuery modelName={modelName}>
     {({
       state: queryState,
-      modelImages = modelImageProcessor(queryState.model ? queryState.model.files.hits.edges : []),
+      modelImages = modelImageProcessor(
+        queryState.model && queryState.model.files && queryState.model.files.hits
```

Comment:
if you want you can use 

```
queryState?.model?.files?.hits?.edges || []
```

Reply:
Yeah looking forward to optional chaining once it's supported out of the box in "create-react-app", right now it's just more convenient to write it the old fashioned way (at least more me).